### PR TITLE
Use pion/webrtc for server-webrtc test code

### PR DESCRIPTION
**Pull request by cohosh**
This removes go-webrtc as a dependency for all of Snowflake.

### CODE REVIEW

File: server-webrtc/http.go [hunk 1]
```diff
@@ -72,3 +73,49 @@ func receiveSignalsHTTP(addr string, config *webrtc.Configuration) error {
 	log.Printf("listening HTTP on %s", addr)
 	return http.ListenAndServe(addr, nil)
 }
+
+func deserializeSessionDescription(msg string) *webrtc.SessionDescription {
```
Comment (arlolra) [inline server-webrtc/http.go]: This code will now live in three places: `client/lib/util.go`, `proxy-go/snowflake.go`, and here.  Maybe it belongs under `common/`?

Comment (cohosh) [reply server-webrtc/http.go]: Hm, yeah. I'd say we should definitely do this if we're going to maintain server-webrtc. If not, then we could still do it but I'd prefer to make that a separate refactor ticket.

**Comment by cohosh** [comment]
Let's just remove this code instead.

File: server-webrtc/http.go (hunk 1 above)
Comment (arlolra) [reply server-webrtc/http.go]: Sounds good

Comment (arlolra) [reply server-webrtc/http.go]: I took a stab at this here,
https://github.com/keroserene/snowflake/commit/7b761d4c8d0e56b9148f106eb01667a7ec5c0424

**Pull request closed without merging**